pkg/sql/codegen/pai/template_xgboost.go

```diff
@@ -153,7 +169,11 @@ model_params,
 train_params,
 feature_metas,
 feature_column_names,
-label_meta) = model.load_metas("{{.OSSModelDir}}", "xgboost_model_desc")
+label_meta,
+feature_columns_code) = model.load_metas("{{.OSSModelDir}}", "xgboost_model_desc")
+
+feature_column_transformers = eval('[{}]'.format(feature_column_code))
+transform_fn = xgboost_extended.feature_column.ComposedColumnTransformer(feature_column_names, *feature_column_transformers)
```

Comment:
Please add a description here that in the current implementation, we are generating an transform_fn from COLUMN expression. The transform_fn is executed during the process of dumping the original data into DMatrix.

Reply:
Done. Please see [here](https://github.com/sql-machine-learning/sqlflow/pull/2361/files#diff-d9418c7b645edf2bcca1f77a708f70d5R98).